web/src/reducers/apd.test.js

```diff
@@ -751,7 +751,7 @@ describe('APD reducer', () => {
                     }
                   }
                 ],
-                fundingSource: 'HIT',
+                fundingSource: false,
```

Comment:
It feels a little weird to set this to false and then set it to a string later on. Would it make more sense to make this an empty string? (I realize then we'd have to do a `if (fundingSource !== '')` later, which isn't super nice.) Just thought I'd flag it!

Reply:
Thankfully we could still do a truthy check because the empty string is falsey. I waffled on this exact question for a bit before deciding that `false` felt like it more strongly indicated that no value was set. Though now that I type that out, `null` is probably the even better answer.

Reply:
Let's talk about this again in parking lot time next week. I'm going to go ahead and merge the PR but I agree we should revisit this in particular for code consistency's sake.